### ISSUE TITLE
Classes and Top of file Docstrings

### ISSUE THREAD

**Issue by Zarad1993** [Zarad1993/dyc#5]
Docstrings are missing in new clases and at the top of the file

**Comment by frangiz** [comment]
Hey, I can fix this if you want. Will fix a PR for you.

**Comment by Zarad1993** [comment]
@frangiz - That'll be great. Thanks

**Comment by demern** [comment]
Hey there--is this still an issue that you'd like folks to help out with? I see that a pull request was merged about a year ago, but looking at the project today I still see a bunch of methods in the 'dyc' classes themselves that are missing docstrings and getting picked up by the tool:

```
(dyc-venv) [ndemers@localhost dyc]$ dyc start
`dyc.yaml` Missing or Incorrectly formatted. USING default settings

Processing Methods



In file dyc/dyc/utils.py :

Do you want to document method list keywords: list of keywords like for python, func for go etc.? [y/N]: 


In file dyc/tests/test_utils.py :

Do you want to document method test_should_return_none_if_not_found? [y/N]: 
Do you want to document method test_tabs? [y/N]: 
Do you want to document method test_2_spaces? [y/N]: 
Do you want to document method test_falsy_value? [y/N]: 
Do you want to document method test_default_4_spaces? [y/N]: 
Do you want to document method test_existing_extension_valid? [y/N]: 
Do you want to document method test_non_existing_extension? [y/N]: 
Do you want to document method test_wrong_extension_type? [y/N]: 
(dyc-venv) [ndemers@localhost dyc]$ 
```

If so, let me know and I can take a stab at adding docstrings for the above methods so `dyc` gives clean output for the project sources.

**Comment by Zarad1993** [comment]
Hey @demern - Please go ahead with it. I don't think anyone is looking at this currently.

**Comment by Zarad1993** [comment]
> If so, let me know and I can take a stab at adding docstrings for the above methods so dyc gives clean output for the project sources.

Good idea

**Comment by demern** [comment]
Hi @Zarad1993 - I've posted a PR for the new docstrings here: https://github.com/Zarad1993/dyc/pull/61

I also filed a new issue for another bug I observed along the way: https://github.com/Zarad1993/dyc/issues/62

**Comment by Zarad1993** [comment]
Thanks @demern - I will try to move the PR forward today/tomorrow max.
Your contribution is **highly** appreciated.

**Comment by AshirGuptash** [comment]
Hi I'd like to contribute to this (for the classesBuilder() part)

**Comment by Zarad1993** [comment]
Yes, perhaps the parameters of the class inheritance would work perfectly.. We can iterate on it if you want too.

**Comment by AshirGuptash** [comment]
Sounds good, we can start with just inheritance arguments in the docstring, and add to afterward.

**Comment by AshirGuptash** [comment]
Hi @Zarad1993 , we sent a PR #68 . Can you review it when you get the chance? Thanks!

**Comment by gkuznik** [comment]
@Zarad1993 This issue should be closed, right?
